### PR TITLE
Support multi yaml files

### PR DESCRIPTION
**Pull request by jiaoew1991**
issue: https://github.com/milvus-io/milvus/issues/22556

### CODE REVIEW

File: internal/util/paramtable/base_table.go [hunk 1]
```diff
@@ -81,7 +81,7 @@ func NewBaseTableFromYamlOnly(yaml string) *BaseTable {
 // GlobalInitWithYaml should be called only in standalone and embedded Milvus.
 func (gp *BaseTable) GlobalInitWithYaml(yaml string) {
 	gp.once.Do(func() {
-		defaultYaml = yaml
+		defaultYaml = []string{yaml}
```
Comment (congqixia) [inline internal/util/paramtable/base_table.go:84]: Shall we support multiple yaml files here?

File: internal/util/paramtable/base_table.go [hunk 2]
```diff
@@ -110,13 +110,14 @@ func (gp *BaseTable) init(refreshInterval int) {
 
 func (gp *BaseTable) initConfigsFromLocal(refreshInterval int) {
 	gp.configDir = gp.initConfPath()
-	configFilePath := gp.configDir + "/" + gp.YamlFile
 	err := gp.mgr.AddSource(config.NewFileSource(&config.FileInfo{
-		Filepath:        configFilePath,
+		Files: lo.Map(gp.YamlFiles, func(file string, _ int) string {
+			return gp.configDir + "/" + file
```
Comment (congqixia) [inline internal/util/paramtable/base_table.go:115]: `path.Join`

File: internal/config/source_test.go [hunk 3]
```diff
@@ -17,20 +17,35 @@
...
 		err := fs.loadFromFile()
 		assert.Error(t, err)
 	})
+
+	t.Run("multiple files", func(t *testing.T) {
+		dir, _ := os.MkdirTemp("", "milvus")
+		os.WriteFile(dir+"/milvus.yaml", []byte("a.b: 1\nc.d: 2"), 0600)
+		os.WriteFile(dir+"/user.yaml", []byte("a.b: 3"), 0600)
+
+		fs := NewFileSource(&FileInfo{[]string{dir + "/milvus.yaml", dir + "/user.yaml"}, -1})
```
Comment (congqixia) [inline internal/config/source_test.go:44]: `path.Join`

File: internal/util/paramtable/base_table.go (hunk 1 above)
Comment (jiaoew1991) [reply internal/util/paramtable/base_table.go:84]: yes, keep the API simple

File: internal/util/paramtable/base_table.go (hunk 2 above)
Comment (jiaoew1991) [reply internal/util/paramtable/base_table.go]: done

File: internal/config/source_test.go (hunk 3 above)
Comment (jiaoew1991) [reply internal/config/source_test.go]: done

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**